python/rmm/rmm/pylibrmm/memory_resource.pyx

```diff
@@ -203,6 +205,45 @@ cdef class CudaAsyncMemoryResource(DeviceMemoryResource):
         )
 
 
+cdef class CudaAsyncViewMemoryResource(DeviceMemoryResource):
+    """
+    Memory resource that uses ``cudaMallocAsync``/``cudaFreeAsync`` for
+    allocation/deallocation with an existing CUDA memory pool.
+
+    This resource uses an existing CUDA memory pool handle (such as the default pool)
+    instead of creating a new one. This is useful for integrating with existing GPU
+    applications that already use a CUDA memory pool, or customizing the flags
+    used by the memory pool.
+
+    Parameters
+    ----------
+    valid_pool_handle : cudaMemPool_t
```

Comment:
I don't like the name `valid_pool_handle` but I copied it from C++. Should we consider renaming it?

Reply:
Sure, though please mention somewhere in this docstring that the user is responsible for keeping the mempool alive

Reply:
Done in b7e899d62127f06b6729ed1426869a255a4fa539. The new sentence says,

> The memory pool passed in must not be destroyed during the lifetime of this memory resource.